### PR TITLE
Use `flutter pub add` to add dependencies

### PR DESCRIPTION
**Pull request by domesticmouse**
PTAL @kevmoo

### CODE REVIEW

**Comment by kevmoo** [comment]
@domesticmouse – there's a kinda weird test that validates the contents of the readme you'll want to check!

**Review by kevmoo — CHANGES_REQUESTED** [review]
Let's drop this change. I want to be clear on the versions needed, etc. It matters.

Maybe add a note after the existing block saying one can use `dart pub add` etc to add the deps?

**Comment by domesticmouse** [comment]
> I want to be clear on the versions needed, etc. It matters.

ELI5 time: 
  - Why should a normal developer care about exact versions, as opposed to using the latest usable version as `dart pub add` does automatically? 
  - What does the developer gain by taking these additional step? 
  - What do we gain from this required maintenance?
